Dockerfile

```diff
@@ -34,7 +35,7 @@ ENV SIMDATA_DIR=/var/simdata
 
 # Download
 RUN apt-get update \
-    && apt-get install -y ca-certificates curl libx11-6 libexpat1 python3 python3-pip \
+    && apt-get install -y ca-certificates curl libx11-6 libexpat1 python3 python3-pip $(["$UBUNTU_BASE" = "22.04"] && echo -n "libmd0") \
```

Comment:
You wanna leave this 10 minute effort in here for the future I suppose?

Reply:
Yep, until we determine it's not necessary.  And the conditional base image builds work really well